### PR TITLE
Add fhir-tooling documentation to the docusite

### PR DESCRIPTION
**Pull request by sharon2719**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes [link to issue]

**Engineer Checklist**
- [x] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Comment by pld** [comment]
I think you're using an older version of docusaurus locally, this doesn't run for me, let me make some updates...

File: docs/engineering/tooling/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# fhircore-tooling
+
+A repo to hold our FHIR content and configuration creation tools and scripts.
```
Comment (pld) [inline docs/engineering/tooling/readme.mdx:3]: Please change to be more descriptive of the content in this section

File: docs/engineering/tooling/readme.mdx [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+- [cleaner](https://github.com/onaio/fhircore-tooling/tree/main/cleaner)
+- [efsity](https://github.com/onaio/fhircore-tooling/tree/main/efsity)
+- [importer](https://github.com/onaio/fhircore-tooling/tree/main/importer)
+- [uploader](https://github.com/onaio/fhircore-tooling/tree/main/uploader)
```
Comment (pld) [inline docs/engineering/tooling/readme.mdx:8]: I think all these links should be removed, we can add brief descriptions of each tool and link to the docs sub-page

File: docs/engineering/tooling/uploader.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# uploader script
```
Comment (pld) [inline docs/engineering/tooling/uploader.mdx:1]: @Wambere does efsity handle uploading yet or are people still using this script?

File: docs/engineering/tooling/efsity.mdx [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+-o or --output - the output path, can be a file or directory. Optional - default is current directory
+-sm or --structure-maps - the directory path to the location of structure map .txt or .map files. Optional - Must be a directory. Must be used with the -q flag
+-q or --questionnaires - the directory path to the location of questionnaires .json files. Optional - Must be a directory. Must be used with the -sm flag
+```
+
+**Note:** To include _Questionnaire_ and _Structure Map_ validation add the `-sm` and `-q` flags
+
+**Sample screenshot output**
+
+<img width="715" alt="Screenshot 2023-03-27 at 21 43 09" src="https://user-images.githubusercontent.com/10017086/228037581-209f9bab-d1b9-45eb-a920-aa12c70c5b98.png"/>
```
Comment (pld) [inline docs/engineering/tooling/efsity.mdx:130]: move image file to a new file in this PR and point to that file here

**Review by pld — CHANGES_REQUESTED** [review]
great start! made some suggestions before we get this merged

File: docs/engineering/tooling/uploader.mdx (hunk 3 above)
Comment (Wambere) [reply docs/engineering/tooling/uploader.mdx:1]: I think some people are still using the script, but the uploader is ready for use. I'll set up a session for a walk-through just so everyone is aware

**Comment by pld** [comment]
> @sharon2719 now that the tooling repo is OSS I think we should maintain the documentation in the repositories and only have the overviews and corresponding links to the actual repo/docs for easier management (and to avoid duplication) cc @pld @Wambere

yea, I agree, @sharon2719 let's remove everything except the readme

File: docs/engineering/tooling/readme.mdx [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+- [**cleaner**](https://github.com/onaio/fhir-tooling/blob/a2f435fce32f029260f8ecda81f41703b4b05456/cleaner/README.md) - it efficiently removes specific FHIR resources from a FHIR API based on user-defined filtering criteria, streamlining data maintenance.
+- [**efsity**](https://github.com/onaio/fhir-tooling/blob/2c94aeeb54a39964cc5ba158ec077e2d1f73564e/efsity/README.md) - efsity streamlines authoring and managing FHIRCore app configurations and content through command-line conversions, validations, translations, and publishing.
+- [**importer**](https://github.com/onaio/fhir-tooling/blob/a2f435fce32f029260f8ecda81f41703b4b05456/importer/README.md) - The importer script streamlines the management of FHIR resources and Keycloak roles through CSV-based bulk operations, covering creation, assignment, updates, and even deletion of duplicates.
+- [**uploader**](https://github.com/onaio/fhir-tooling/blob/a2f435fce32f029260f8ecda81f41703b4b05456/importer/README.md) - The uploader script automates content uploads to a FHIR server, simplifying the process through command-line execution and configuration.
```
Comment (pld) [inline docs/engineering/tooling/readme.mdx:8]: please link to the main branch, not a commit hash

**Review by pld — CHANGES_REQUESTED** [review]

**Review by pld — APPROVED** [review]

**Pull request merged**